apps/ledger-live-mobile/src/screens/WalletCentricAsset/AssetRow.tsx

```diff
@@ -1,15 +1,36 @@
 import React, { useCallback } from "react";
 import useEnv from "@ledgerhq/live-common/hooks/useEnv";
 import { BigNumber } from "bignumber.js";
+import { CryptoOrTokenCurrency } from "@ledgerhq/types-cryptoassets";
+import { AccountLike } from "@ledgerhq/types-live";
 import { NavigatorName, ScreenName } from "../../const";
-import { usePortfolio } from "../../actions/portfolio";
+import { usePortfolio } from "../../hooks/portfolio";
 import AssetRowLayout from "../../components/AssetRowLayout";
 import { track } from "../../analytics";
+import {
+  BaseNavigationComposite,
+  StackNavigatorNavigation,
+} from "../../components/RootNavigator/types/helpers";
+import { AccountsNavigatorParamList } from "../../components/RootNavigator/types/AccountsNavigator";
+import { PortfolioNavigatorStackParamList } from "../../components/RootNavigator/types/PortfolioNavigator";
+
+type Asset = {
```

Comment:
todo : create a common Asset type @nparigi-ledger @cgrellard-ledger @mlegall-ledger 
It's used in multiple component

Reply:
doing

Reply:
done